Source/Core/Core/NetPlayServer.cpp

```diff
@@ -173,7 +173,7 @@ void NetPlayServer::ThreadFunc()
             delete (PlayerId*)netEvent.peer->data;
             netEvent.peer->data = nullptr;
           }
-          enet_peer_disconnect(accept_peer, 0);
```

Comment:
Did you intend to include this change in the PR?

Reply:
Yeah, `enet_peer_disconnect` closes the connection immediately, while `enet_peer_disconnect_later` waits for all remaining queued packets to be sent. Without this change, the client never actually receives the error message packet.